environment/src/test/java/com/sequenceiq/environment/environment/validation/validators/TagValidatorTest.java

```diff
@@ -91,6 +94,12 @@ PlatformParameters parameters() {
             when(mock.tagValidator()).thenReturn(azureTagValidator);
             return mock;
         }
+
+        @Bean
+        @Qualifier("AzureImageFilter")
+        ImageFilter azureImageFilter() {
+            return new CommonImageFilter();
```

Comment:
no assert or mock on this so probably more test must be added.

Reply:
same as above